fossunited/doctype_ids.py

```diff
@@ -7,7 +7,7 @@
 HACKATHON = "FOSS Hackathon"
 JOIN_TEAM_REQUEST = "FOSS Hackathon Join Team Request"
 HACKATHON_LOCALHOST = "FOSS Hackathon LocalHost"
-LOCALHOST_ORGANIZER = "FOSS Hackathon Localhost Organizer"
+LOCALHOST_ORGANIZER = "FOSS Hackathon LocalHost Organizer"
```

Comment:
i guess this wasn't a problem for us because this refactor happened after FOSS Hack 3.0 ended, right?

Reply:
Yeah, we did the whole doctype ID thing after FOSS Hack